tools/c7n_azure/c7n_azure/resources/vm.py

```diff
@@ -31,3 +31,15 @@ class resource_type(object):
             'resourceGroup',
             'properties.hardwareProfile.vmSize',
         )
+
+@VirtualMachine.filter_registry.register('instance-view')
+class InstanceViewFilter(ValueFilter):
+    schema = type_schema('instance-view', rinherit=ValueFilter.schema)
+
+    def __call__(self, i):
+        if 'instanceView' not in i:
```

Comment:
I would prefer a more descriptive variable name here than "i".

Reply:
I think `i` is used in every filter implementation (it's an interator and `i` is the current item)